content/documentations/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Documentations for Apache Camel
+---
+
```

Comment:
I think we need an introduction here on how to find the right documentation and how to contribute to the documentation.

Reply:
I hope the changes are better now!